Sources/ProjectAutomation/Tuist.swift

```diff
@@ -34,7 +34,7 @@ public enum Tuist {
             var arguments = [
                 "tuist",
                 "graph",
-                "--format", "json",
+                "--format", "--legacyJSON",
```

Comment:
@fortmarek is this correct value `--legacyJSON` or should it be `legacyJSON` (without hyphens/dashes)?